src/snyk/cli/process.ts

```diff
@@ -25,6 +25,7 @@ export class CliProcess {
     return new Promise((resolve, reject) => {
       let output = '';
 
+      // file deepcode ignore ArrayMethodOnNonArray: <please specify a reason of ignoring this>
```

Comment:
What's the reason for that?

Reply:
this was a false positive as `args` is an array of strings. I've updated the ignore with the reason